lockbox-ios/Store/UserDefaultStore.swift

```diff
@@ -60,3 +73,19 @@ class UserDefaultStore: BaseUserDefaultStore {
         }
     }
 }
+
+extension UserDefaultStore {
+    private func readValuesFrom(_ userDefaults: UserDefaults) {
+        for key in LocalUserDefaultKey.allValues {
```

Comment:
This is effectively moving everything from the "old" UserDefaults to the shared one, yes?

Is there a point in keeping the `LocalUserDefaultKey` separate from `UserDefaultKey`?

Reply:
The general idea is to keep settings that aren't relevant to the credential provider isolated to the application. For settings like `preferredBrowser` etc, we don't need to include the`Setting.PreferredBrowser` class in the credential provider. The impetus for keeping the keys separated as well is the `defaultValue` parameter required for all `UserDefaultKey` values.